scripts/install_epinio.sh

```diff
@@ -66,6 +66,9 @@ helm upgrade --debug --wait --install -n epinio --create-namespace epinio ${CHAR
 # Wait for Epinio deployment to be ready
 kubectl rollout status deployment epinio-server -n epinio --timeout=480s
 
+# Temporary fix for unavailable epinio-unpacker:v1.10.0-rc1 image
```

Comment:
Maybe would be nice to add a comment suggestion to keep it commented unless a problem in deployment type in https://github.com/epinio/epinio/issues/2583 appears: 
```
AssertionError: Timed out retrying after 240000ms: Expected to find content: '===> EXPORTING' but never did.
```
or

```
docker pull ghcr.io/epinio/epinio-unpacker:v1.10.0-rc1
Error response from daemon: manifest unknown
```